build.sbt

```diff
@@ -2,35 +2,33 @@ import ReleaseTransformations._
 
 name := "akka-xlsx"
 
-lazy val AkkaVersion        = "2.5.19"
-lazy val AkkaContribVersion = "0.9"
-lazy val AlpakkaXmlVersion  = "0.20"
+lazy val AkkaVersion        = "2.5.31"
```

Comment:
I'm curious about the reasoning:
- not to go `2.5.32` if performing only bugfix upgrade
- not to go `2.6.10`

Reply:
What I can say... Yesterday I had really good reason (version conflict indication). Today I cannot find it :)